src/mvBasicWidgets.cpp

```diff
@@ -4743,9 +4745,16 @@ DearPyGui::draw_listbox(ImDrawList *drawlist, mvAppItem &item, mvListboxConfig &
 
         if (ImGui::ListBox(item.info.internalLabel.c_str(), item.config.enabled ? &config.index : &config.disabledindex, config.charNames.data(), (int)config.names.size(), config.itemsHeight))
         {
-            *config.value = config.names[config.index];
-            config.disabled_value = config.names[config.index];
-            auto value = *config.value;
+			if (config.sendindex) {
+				*config.value = std::to_string(config.index);
```

Comment:
I don't really like the idea of converting the index to a string here. I think `mvListbox::getPyValue()` should handle it and return a string if `index_items` is False, and a number (the index) if it's True. The same for the callback: instead of converting the index to a string, it should call `ToPyInt` and return an integer.